etl/steps/data/garden/ahdi/2023-09-08/augmented_hdi.meta.yml

```diff
@@ -0,0 +1,294 @@
+
+          If we denote the non-linearly transformed values of life expectancy and education as LEB and EDU, LD to represent those of liberal democracy, and the adjusted per capita income as UNY, the historical index of human development can be expressed as,
+
+          AHDI = (LEB * EDU * LD * UNY)^1/4^
+        processing_level: minor
+        description_key:
+          - "{tables.augmented_hdi.variables.ahdi.description_short}"
+          - "Each of these dimensions are measured as sub-indices:"
+          - Life expectancy is the proxy for healthy life, where 1 represents 85 or more years and 0 represents 20 or less years.
+          - Years of schooling is the proxy for access to knowledge, where 1 represents 15 or more years of education and 0 represents 0 years of education.
+          - Liberal democracy is the proxy for freedom, it ranges from 0 to 1, where higher values represents more democracy.
+          - Adjusted (log) GDP per capita is the proxy for wellbeing dimensions different from education and health, bounded between 47,000 and 100 international-$ at 1990 prices.
+          - The AHDI is then estimated as the geometric of these indices, or
+            AHDI = (LEB * EDU * LD * UNY)^1/4^
+            Where LEB is life expectancy, EDU is years of schooling, LD is liberal democracy and UNY is adjusted GDP per capita.
+        display:
+          name: AHDI
+          entityAnnotationsMap: "{common.entity_annotations}"
+          numDecimalPlaces: 2
+          conversionFactor: 1
+        presentation:
+          grapher_config:
+            slug: augmented-human-development-index
+            title: Augmented Human Development Index
+            subtitle: >-
+              The Augmented Human Development Index (AHDI) is a summary measure of
+              average achievement in four key dimensions of human development: a long and
+              healthy life, being knowledgeable, being free and having a decent standard of living.
+            hasMapTab: true
+            tab: map
+            originUrl: https://ourworldindata.org/human-development-index
+            yAxis:
+              min: 0
+            map:
+              time: latest
+              colorScale:
+                baseColorScheme: YlGnBu
+                binningStrategy: manual
+                customNumericValues:
+                  - 0.1
+                  - 0.2
+                  - 0.3
+                  - 0.4
+                  - 0.5
+                  - 0.6
+                  - 0.7
+                  - 0.8
+                  - 0.9
+            selectedEntityNames:
+              - United Kingdom
+              - United States
+              - South Korea
+              - India
+              - China
+              - Brazil
+              - Cape Verde
+            $schema: https://files.ourworldindata.org/schemas/grapher-schema.003.json
+
+          topic_tags_links:
```

Comment:
This field has been renamed `topic_tags`, but maybe we can refactor these things in a separate PR.

Reply:
(My bad, I was helping PabloA with renaming and forgot this one. @paarriagadap can you rename it please?)